src/Tag/TagHandler.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+/*
+ * This file is part of the FOSHttpCache package.
+ *
+ * (c) FriendsOfSymfony <http://friendsofsymfony.github.com/>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace FOS\HttpCache\Tag;
+
+use FOS\HttpCache\Exception\UnsupportedProxyOperationException;
+use FOS\HttpCache\ProxyClient\ProxyClientInterface;
+use FOS\HttpCache\ProxyClient\Invalidation\BanInterface;
+
+/**
+ * Handler for cache tagging.
+ *
+ * @author David de Boer <david@driebit.nl>
+ * @author David Buchmann <mail@davidbu.ch>
+ */
+class TagHandler
```

Comment:
i added no flush method to this class. this is fine when using the same cache proxy client as for the CacheInvalidator, but would not work with separate client instances. but it should not be the job of a TagHandler to flush the cache client imho.